.github/workflows/build_doc.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         run: |
           # install doxygen
           wget https://www.doxygen.nl/files/doxygen-$DOXY_VER.linux.bin.tar.gz
-          tar -xzf doxygen-$DOXY_VER.linux.bin.tar.gz
+          tar -I pigz -xf doxygen-$DOXY_VER.linux.bin.tar.gz
```

Comment:
this job is running on default docker image. Do you need to install this command separately?

Reply:
no, gha images already includes it (https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md)